tests/process_monitoring/test_critical_process_monitoring.py

```diff
@@ -522,7 +522,46 @@ def ensure_all_critical_processes_running(duthost, containers_in_namespaces):
                     ensure_process_is_running(duthost, container_name_in_namespace, program_name)
 
 
-def test_monitoring_critical_processes(duthosts, rand_one_dut_hostname, tbinfo, skip_vendor_specific_container):
+def get_skip_containers(tbinfo, skip_vendor_specific_container):
+    skip_containers = []
+    skip_containers.append("database")
+    skip_containers.append("gbsyncd")
+    # Skip 'restapi' container since 'restapi' service will be restarted immediately after exited,
+    # which will not trigger alarm message.
+    skip_containers.append("restapi")
```

Comment:
>append [](http://example.com/codeflow?start=20&length=6)

do you want to skip acms container? public image has no acms container.

Reply:
I discussion this with gang, I will add skip acms code in internal repo.